nautobot_circuit_maintenance/tests/test_handler.py

```diff
@@ -134,7 +134,6 @@ def test_run_simple(self):
             self.assertEqual(1, len(ParsedNotification.objects.all()))
             self.assertEqual(1, len(CircuitMaintenance.objects.all()))
             self.assertEqual(2, len(CircuitImpact.objects.all()))
-            self.assertEqual(0, len(Note.objects.all()))
```

Comment:
Can we change this to `Note.objects.filter(assigned_object_type=ContentType.objects.get_for_model(CircuitMaintenance))` or similar, rather than removing this check?

Similar comment throughout this file.

Reply:
good one

Reply:
@chadell planning to address this?

Reply:
also forgot 🤦